CHANGELOG.md

```diff
@@ -3,14 +3,248 @@
+    new object-centric APIs with a more scalable, consistent, and ergonomic experience.
+   - Bringup of C++ support is now underway and will eventually become our third officially supported SDK language.
+
+### Known Regressions
+- Due to the Blueprint storage migration, blueprint persistent on web is currently broken. Will be resolved in: [#2579][https://github.com/rerun-io/rerun/issues/2579]
+
+### In Detail
+#### 🐍 Python SDK
+- Clean up warnings printed when `rr.init` hasn't been called [#2209](https://github.com/rerun-io/rerun/pull/2209)
+- Normalise Python typing syntax to 3.8+ [#2361](https://github.com/rerun-io/rerun/pull/2361)
+- Simpler, sturdier stateful time tracking in both SDKs [#2506](https://github.com/rerun-io/rerun/pull/2506)
+- Fix not taking np.array for single colors [#2569](https://github.com/rerun-io/rerun/pull/2569)
+- Expose batch APIs for linestrips [#2822](https://github.com/rerun-io/rerun/pull/2822)
+- Expose batch APIs for oriented bounding boxes [#2823](https://github.com/rerun-io/rerun/pull/2823)
+- Improve `log_pinhole` and support non-RDF pinholes [#2614](https://github.com/rerun-io/rerun/pull/2614)
+
+#### 🦀 Rust SDK
+- Add example for adding custom Space Views [#2328](https://github.com/rerun-io/rerun/pull/2328)
+- Simpler, sturdier stateful time tracking in both SDKs [#2506](https://github.com/rerun-io/rerun/pull/2506)
+- Automagic flush when `take()`ing a `MemorySinkStorage` [#2632](https://github.com/rerun-io/rerun/pull/2632)
+- Logging SDK: Log warnings if user data is dropped [#2630](https://github.com/rerun-io/rerun/pull/2630)
+- Add support for `RecordingStream::serve` [#2815](https://github.com/rerun-io/rerun/pull/2815)
+
+#### 🌁 Viewer Improvements
+- Better handle scroll-to-zoom in 3D views [#1764](https://github.com/rerun-io/rerun/pull/1764)
+- Add command to screenshot the application [#2293](https://github.com/rerun-io/rerun/pull/2293)
+- Show layout in blueprint tree view [#2465](https://github.com/rerun-io/rerun/pull/2465)
+- Double-click to select entity [#2504](https://github.com/rerun-io/rerun/pull/2504)
+- Add Rerun.io link/text in top bar [#2540](https://github.com/rerun-io/rerun/pull/2540)
+- New auto-layout of space views [#2558](https://github.com/rerun-io/rerun/pull/2558)
+- Add 'Dump datastore' command to palette [#2564](https://github.com/rerun-io/rerun/pull/2564)
+- Support any `dtype` for depth images [#2602](https://github.com/rerun-io/rerun/pull/2602)
+- Change "Save Selection" command to Cmd+Alt+S [#2631](https://github.com/rerun-io/rerun/pull/2631)
+- Consistent transform visualization for all entities with transforms [#2577](https://github.com/rerun-io/rerun/pull/2577)
+- Improve `log_pinhole` and support non-RDF pinholes [#2614](https://github.com/rerun-io/rerun/pull/2614)
+
+#### 🚀 Performance Improvements
+- Flush the batches every 8ms instead of 50 ms [#2220](https://github.com/rerun-io/rerun/pull/2220)
```

Comment:
I feel like this deserves to be mentioned in the overview. This really is more than a performance improvement, it really affects the usability of the viewer in real-time scenarios...

It's also a good opportunity to quickly remind users of micro-batching, that can't hurt.

Something like:
> The Rerun SDK now defaults to 8ms long microbatches instead of 50ms. This makes the default behavior much more suitable for e.g. real-time video feeds.
> Check out [the microbatching docs](https://www.rerun.io/docs/reference/sdk-micro-batching) for more information.

But better :smile: